auction-server/src/auction.rs

```diff
@@ -1082,6 +1086,10 @@ impl PartialEq<SimulatedBidSvm> for BidSvm {
     }
 }
 
+const BID_MINIMUM_LIFE_TIME_SVM_SERVER: i64 = 5;
+const BID_MINIMUM_LIFE_TIME_SVM_OTHER: i64 = 10;
+const BID_MINIMUM_LIFE_TIME_SVM_INVALID: i64 = 0;
```

Comment:
maybe a function that gets a SubmitType and returns `Option<i64>` where None means do not check for lifetime